content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -30,11 +30,15 @@ Here is an example of what this file might look like:
 ```
 apiVersion: kubelet.config.k8s.io/v1beta1
 kind: KubeletConfiguration
+address: "0.0.0.0",
+port: 10250,
```

Comment:
Em ... aren't these the defaults?

Reply:
yes, they are the defaults and only serve the purpose of showing how to specify simple string/int/bool parameters. I have updated them to non-default values to avoid confusion.